packages/gator-permissions-snap/src/utils/validate.ts

```diff
@@ -17,3 +19,14 @@ export const validatePermissionRequestParam = (
 
   return validateGrantAttenuatedPermissionsParams.data;
 };
+
+/**
+ * Zod validation for startTime to ensure it's today or later.
+ * @param value - Unix timestamp in seconds.
+ * @returns True if the start time is today or later, false otherwise.
+ */
+export const validateStartTimeZod = (value: number): boolean => {
+  // TODO: change to getStartOfTodayLocal when dateTime gets merged.
```

Comment:
should we capture this in a ticket, or do you just want to sequence those two PRs so that we can update this PR before merging?